### PR TITLE
Creation modal view controller tests

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Tests for creation modal view controller and some general things like importing powermockito

### CODE REVIEW

File: src/main/java/control/CreationModalViewController.java [hunk 1]
```diff
@@ -116,6 +120,7 @@ private void cameraShotStartCountFocusHandler(ObservableValue<? extends Boolean>
      * @param event - the event to handle
      */
     private void createCameraShot(MouseEvent event) {
+        System.out.println(validateCameraShot());
```
Comment (Markvanderuit) [inline src/main/java/control/CreationModalViewController.java]: print statement here.

File: src/main/java/gui/modal/ModalView.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public ModalView(RootPane rootPane, int width, int height) {
         this.height = height;
         this.modalStage = new Stage();
         this.modalStage.initModality(Modality.APPLICATION_MODAL);
-        this.modalStage.initOwner(rootPane.getPrimaryStage());
+//        this.modalStage.initOwner(rootPane.getPrimaryStage());
```
Comment (geenen124) [inline src/main/java/gui/modal/ModalView.java]: Why is this commented out?

Comment (Markvanderuit) [inline src/main/java/gui/modal/ModalView.java]: unused code?

Comment (bart-de-jonge) [inline src/main/java/gui/modal/ModalView.java]: fixed this

**Comment by geenen124** [comment]
Looks good! Merging

**Pull request merged**